frontend/lib/src/components/shared/Modal/Modal.tsx

```diff
@@ -95,7 +95,6 @@ function ModalBody({ children }: Readonly<ModalBodyProps>): ReactElement {
         paddingLeft: spacing.twoXL,
         color: colors.bodyText,
         fontSize: fontSizes.md,
-        overflowY: "auto",
```

Comment:
**note:** I was a little worried that this change would make it such that anything that overflows in the modal body would be unusable. I tried this locally and learned that the entire modal scrolls if necessary, so seems like a reasonable change to me.